### DISCUSSION TITLE
How to display code collapse feature in nb 14 ?

### DISCUSSION THREAD

**Discussion by rfrkk** [apache/netbeans#4283] [General]
Netbaens version used ： 14

Problems encountered：

### How to fix it? Tks.

![Snipaste_2022-06-25_08-13-26](https://user-images.githubusercontent.com/137764/175750757-bbe141fe-3225-4d46-a862-c05d45099c17.png)

![Snipaste_2022-06-25_08-11-52](https://user-images.githubusercontent.com/137764/175750766-d1ee9130-82d8-4efc-87d9-81ad88d98f44.png)

**Comment by neilcsmith-net** [comment]
Should just be there.  Did you get this working?  Does right-click Code Folds / Collapse All work?

If not, please open a bug with a simple example project that shows the issue, and anything that looks relevant in `View / IDE Log`.

**Comment by rfrkk** [comment]
Thank you for spending time answering my questions. 

![Snipaste_2022-07-21_07-50-59](https://user-images.githubusercontent.com/137764/180101788-61b3c526-475a-45a4-b24f-d5623aeb42e2.png)
